src/components/Editor/Runners/PyodideRunner/PyodideRunner.jsx

```diff
@@ -0,0 +1,364 @@
+/* eslint import/no-webpack-loader-syntax: off */
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
⛏️  I'm not a fan of this rule for sure, especially with empty dependency arrays but 2/3 useEffects that are effected can have `handleRun` and `handleStop` in the array without any impact

Reply:
raising just so I can make a TODO ticket 👍